cmd/meroxa/turbine/utils.go

```diff
@@ -280,12 +280,14 @@ func GetGitBranch(appPath string) (string, error) {
 }
 
 func GetTurbineResponseFromOutput(output string) (string, error) {
-	r := regexp.MustCompile("turbine-response: ([^\n]*)")
+	r := regexp.MustCompile("turbine-response: ([^\r\n]*)")
 	match := r.FindStringSubmatch(output)
 	if match == nil || len(match) < 2 {
 		return "", fmt.Errorf("output is formatted unexpectedly")
 	}
```

Comment:
I didn't make these changes but it feels like given what's being done in version.go `GetVersion` function, it feels like this should NOT actually be returning an error. Instead we should properly handle both forms of the response output here rather than relying on the caller to clean things up.